pyhap/characteristic.py

```diff
@@ -158,16 +183,22 @@ def override_properties(self, properties=None, valid_values=None):
             self.properties[PROP_VALID_VALUES] = valid_values
 
         try:
-            self.value = self.to_valid_value(self.value)
+            self.set_value(self.to_valid_value(self.value), should_notify=False)
         except ValueError:
-            self.value = self._get_default_value()
+            self.set_value(self._get_default_value(), should_notify=False)
```

Comment:
`override_properties` shouldn't call `set_value`. It would only trigger unnecessary calls, like the log output.
Just use:
```py
        try:
            self._value = self.to_valid_value(self.value)
        except ValueError:
            self._value = self._get_default_value()
```